base/sparse/higherorderfns.jl

```diff
@@ -10,7 +10,7 @@ import Base.Broadcast: _containertype, promote_containertype,
 
 using Base: front, tail, to_shape
 using ..SparseArrays: SparseVector, SparseMatrixCSC, AbstractSparseVector,
-                      AbstractSparseMatrix, AbstractSparseArray, indtype
+                      AbstractSparseMatrix, AbstractSparseArray, indtype, nnz, nzrange
```

Comment:
@vtjnash Is this a bug or shouldn't we expect the special behavior of Base, Main, and Core to carry through to `eval(:(include))`